docker.go

```diff
@@ -106,24 +106,32 @@ func (d *dockerContainerService) syncContainers(now time.Time) {
 			continue
 		}
 
-		containerIP := container.NetworkSettings.IPAddress
+		var containerIP []string
+		if container.NetworkSettings.IPAddress != "" {
+			containerIP = append(containerIP, container.NetworkSettings.IPAddress)
+		}
+		for network := range container.NetworkSettings.Networks {
+			containerIP = append(containerIP, container.NetworkSettings.Networks[network].IPAddress)
+		}
 		roleArn, iamPolicy, err := getRoleArnFromEnv(container.Config.Env)
 
 		if err != nil {
 			log.Error("Error getting role from container: ", apiContainer.ID, ": ", err)
 			continue
 		}
 
-		log.Infof("Container: id=%s image=%s role=%s", container.ID[:6], container.Config.Image, roleArn)
-
-		containerIPMap[containerIP] = dockerContainerInfo{
-			containerInfo: containerInfo{
-				ID:        container.ID,
-				Name:      container.Name,
-				IamRole:   roleArn,
-				IamPolicy: iamPolicy,
-			},
-			RefreshTime: refreshAt,
+		for _,IPAddress := range containerIP {
```

Comment:
Add space after comma `_, ipAddress` and start variable name with lowercase (e.g. ipAddress)